appveyor.yml

```diff
@@ -0,0 +1,67 @@
+version: 5.1.0.{build}
+image: Visual Studio 2017
+environment:
+  matrix:
+  - DB: SqlServer2008
+    CONNECTION_STRING: Server=(local)\SQL2017;User ID=sa;Password=Password12!;initial catalog=nhibernate;
+  - DB: PostgreSQL
+    CONNECTION_STRING: Host=localhost;Port=5432;Username=postgres;Password=Password12!;Database=nhibernate;Enlist=true;
+  - DB: Firebird
+  - DB: MySQL
+    CONNECTION_STRING: Server=127.0.0.1;Uid=root;Pwd=Password12!;Database=nhibernate;Old Guids=True;
+init:
+    # Required for having windows endlines in sources zip
+    - git config --global core.autocrlf true
+matrix:
+  allow_failures:
+  - DB: MySQL
+build: off
+before_test:
+- ps: |-
+    switch ($env:DB) {
+      'Firebird' {
+        $FireBirdPath = 'C:\firebird'
+        # Install Firebird
+        # Firebird zip is cached, check if we have it, otherwise download it
```

Comment:
For this to be true, appveyor file must include:
```
cache:
    - C:\firebird\firebird.zip
```
As [here](https://github.com/nhibernate/nhibernate-core/pull/1508/commits/47a11ac4a6181edf025a6691d8a7f843a9e76d6d#diff-180360612c6b8c4ed830919bbb4dd459R40).

And it will actually be taken into account only once merged. On a PR, no caching is done by AppVeyor. (Which is better, this avoids having a bad PR tinkering with cached artifacts.)

Reply:
I had troubles installing firebird on appveyor so disabled the cache. It turns out that the service start did not work in pwsh (powershell core) - the pull hanged after that, so I had to switch to ps.

Reply:
I've removed the FB cache check and mentions.